src/components/cylc/tree/TreeItem.vue

```diff
@@ -64,15 +64,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             :isHeld="node.node.isHeld"
             :progress="node.node.progress"
           />
-          <span class="mx-1">{{ node.node.name }}</span>
           <div v-if="!isExpanded" class="node-summary">
             <!-- Task summary -->
             <job
               v-for="(job, index) in node.children"
               v-cylc-object="job.id"
               :key="`${job.id}-summary-${index}`"
-              :status="job.node.state" />
+              :status="job.node.state"
+              style="margin-left: 0.25em;"
```

Comment:
We are using the Vuetify margin-right (`mr`) and margin-left (`ml`) spaces elsewhere. Would that work here too, for consistency? e.g. `ml-1` (`4px`), `ml-2` (`8px`).

Reply:
If we use px as a measurement then this margin won't scale with the font size when it is changed from the user dashboard.

Really surprised vuetify doesn't use relative units like em's rem's, vw, etc, it's pretty essential for responsive and accessible design.

Reply:
They have a whole docs section on accessibility, but I haven't checked if these px values vary for other viewports.

So far I think the base layout and most components (including the tree component) have been using vuetify classes.

If you prefer this em value maybe it'd be worth adding a css class + scss var in our styles? Up to you 👍